src/search/engines/bingimages/bingimages.go

```diff
@@ -197,31 +187,41 @@ func Search(ctx context.Context, query string, relay *bucket.Relay, options engi
 		}
 	})
 
-	localeParam := getLocale(&options)
+  errChannel := make(chan error, 1)
```

Comment:
Why 1? This means only one error can be returned by bingimages.Search(). Same comment applies for everywhere else.